Firmware/LowLevel/src/main.cpp

```diff
@@ -367,7 +367,7 @@ void setup() {
     rp2040.idleOtherCore();
 
 #ifdef USB_DEBUG
-    DEBUG_SERIAL.begin(9600);
+    DEBUG_BEGIN(9600);
 #endif
```

Comment:
Nit-picking here, but I think you can just always call DEBUG_BEGIN, should be a no-op for non-debug (and I think you even had a comment about that in debug.h 😉).

Reply:
Noo, not nit-picking, you're right. Brainlessly replaced DEBUG_SERIAL...
Now that you stub my nose to my own comment :-) I checked the macro to validate my comment :-).
I don't like all the `#ifdef debug` lines in main code, they disturb reading.
Removed but not committed. Will commit with next changes.